### ISSUE TITLE
issue in jl_init_with_image

### ISSUE THREAD

**Issue by armgong** [JuliaLang/julia#10308]
~~~c
DLLEXPORT void jl_init_with_image(const char *julia_home_dir, const char *image_relative_path)
{
    if (jl_is_initialized()) return;
    ......
    jl_eval_string("Base.init_bind_addr(Base.JLOptions())"); // **now .init_bind_addr need jl_options**
    jl_eval_string("Base.init_head_sched()");
    jl_exception_clear();
}
~~~

**Comment by jakebolewski** [comment]
@armgong what is the issue exactly? How `init_bind_addr` works recently changed in the development version.

**Comment by armgong** [comment]
sorry   jl_eval_string("Base.init_bind_addr(ARGS)") should change to jl_eval_string("Base.init_bind_addr(Base.JLOptions())");  because now init_bind_addr need jl_options as paramters

**Comment by jakebolewski** [comment]
Ok thanks for the bug report, I'll push a fix soon.

**Comment by armgong** [comment]
Thanks

**Comment by jakebolewski** [comment]
Fixed by 0727e33fa56ae3533.
